scripts/secnetperf-helpers.psm1

```diff
@@ -334,13 +334,17 @@ function Wait-LocalTest {
 function Get-TestOutput {
     param ($Output, $Metric)
     if ($Metric -eq "latency") {
-        $latency_percentiles = '(?<=\d{1,3}(?:\.\d{1,2})?th: )\d+'
-        return [regex]::Matches($Output, $latency_percentiles) | ForEach-Object {$_.Value}
+        $latency_percentiles = "(?<=\d{1,3}(?:\.\d{1,2})?th: )\d+"
+        $RPS_regex = "Result:\s+(\d+)"
+        $percentiles = [regex]::Matches($Output, $latency_percentiles) | ForEach-Object {$_.Value}
+        $rps = [regex]::Matches($Output, $RPS_regex)
```

Comment:
Should this have this?
```suggestion
        $rps = [regex]::Matches($Output, $RPS_regex) | ForEach-Object {$_.Value}
```

Reply:
I also wonder if the regex is incorrect because the json output has `Result` in the "value"
```json
    {
      "Groups": "Result: 5029 5029",
      "Success": true,
      "Name": "0",
      "Captures": "Result: 5029",
      "Index": 0,
      "Length": 12,
      "Value": "Result: 5029",
      "ValueSpan": null
    },
```

Reply:
Yeah, after looking at the sql output:
```
INSERT INTO Secnetperf_test_runs (Secnetperf_test_ID, Secnetperf_commit, Client_environment_ID, Server_environment_ID, Result, Secnetperf_latency_stats_ID, io, tls)
VALUES ("rps-up-512-down-4000-tcp-0", "4ef81ab8e8ce1831ced17333541148b5ec669be7", 2, 2, Result: 5029, LAST_INSERT_ROWID(), "epoll", "openssl");INSERT INTO Secnetperf_latency_stats (p0, p50, p90, p99, p999, p9999, p99999, p999999)
VALUES (148, 204, 277, 362, 983, 4329, 9160, 9160);
```
It's pretty clear we don't want `Result: 5029` as the value.

Reply:
Weird... I ran it locally without it, and it worked fine.